### DISCUSSION TITLE
forge test -m is error,

### DISCUSSION THREAD

**Discussion by Chujimafa** [Cyfrin/foundry-full-course-cu#1518] [General]
in forge test -m is error, t tried --mt, then it is working. Hat foundry changed it?

**Comment by EngrPips** [comment]
Hello @Chujimafa , glad you found a way out and yeah `-m` is deprecated and the current command is `--mt`

**Reply by mxnwafor** [reply]
seems `--mt` is same as `--match-test`

**Reply by EngrPips** [reply]
Yeah it is the same

**Comment by save-the-planet08** [comment]
that really helped!

**Reply by EngrPips** [reply]
glad it did!
